lib/zip/entry_set.rb

```diff
@@ -49,7 +49,7 @@ def entries
 
     # deep clone
     def dup
-      EntrySet.new(@entry_set.map { |key, value| value.dup })
+      EntrySet.new(@entry_set.map { |_key, value| value.dup })
```

Comment:
Like this ` EntrySet.new(@entry_set.map(&:dup))`?
Not working, a lot of test failed